lib/ios/UIViewController+LayoutProtocol.h

```diff
@@ -16,7 +16,7 @@ typedef void (^RNNReactViewReadyCompletionBlock)(void);
 
 - (void)mergeOptions:(RNNNavigationOptions *)options;
 
-- (void)mergeChildOptions:(RNNNavigationOptions *)options;
```

Comment:
Isn't `mergeChildOptions` a better name for this function? It's explicit and it's clear how this function differs from the regular `mergeOptions` function.